src/templates/speciality.js

```diff
@@ -47,16 +690,16 @@ const Speciality = ({
   },
   location
 }) => {
-  const {
-    relatedProjects,
-    title,
-    clients,
-    communityBackground,
-    communityLogo,
-    communityText,
-    eventIcon,
-    contactText
-  } = speciality
+  const { relatedProjects, clients, eventIcon } = speciality
```

Comment:
Following on from [these comments](https://github.com/yldio/yld.io/pull/278/files/d258e94bde8561ed06734668c5159e2106242fe3#r299785181)
I would argue this should be placed in the `flattenSpeciality` method.

Reply:
OK, done.